### DISCUSSION TITLE
about 'NoneType' object

### DISCUSSION THREAD

**Discussion by codingaowu** [adamreeve/npTDMS#271] [Q&A]
Hello everyone, I want to ask a question. For excample I have a tdms file include 'NoneType' data in channel , when I read the data first from original tdms file then I want to rewrite them to a new tdms file, got a error message AttributeError: 'NoneType' object has no attribute 'enum_value'. Can I reslove this issue by any method? Thanks

**Comment by adamreeve** [comment]
Hi @bigwolf0, that sounds like a bug in npTDMS, you shouldn't be getting data channels with the type set to None. Are you able to create an issue and provide an example file and code that has this problem?
